app/models/user.rb

```diff
@@ -3,6 +3,8 @@
 class User < ApplicationRecord
   authenticates_with_sorcery!
 
+  enum gender: {other: 0 , man: 1, woman: 2}
```

Comment:
```suggestion
  enum gender: { other: 0 , man: 1, woman: 2 }
```
Rubyでは `{ }` の最初と最後に空白が入っているのが通例なので（動作上問題ないのですが）